experiments/AMIP/moist_mpi_earth/coupler_utils/general_helper.jl

```diff
@@ -0,0 +1,20 @@
+# most of these functions are temporary helpers until upstream issues are resolved
+
+# TODO: unify with coupler interface
+struct CouplerSimulation{I, F, B, T, M}
+    Δt::I
+    t::F
+    boundary_space::B
+    FT::T
+    mask::M
+end
```

Comment:
```suggestion
struct CouplerSimulation{FT, I, F, B, M}
    Δt::I
    t::F
    boundary_space::B
    mask::M
end
```

we can then access `FT` by doing things like `foo(couplersim::CouplerSimulation{FT},  ...) where {FT}`

Reply:
currently am just grabbing FT from eltype(Y), since couplersim isnt used by most methods. but good call for the future!